functional/measured-boot-policy-sanity/test.sh

```diff
@@ -56,27 +56,38 @@ rlJournalStart
         rlRun "limeWaitForAgentRegistration ${AGENT_ID}"
     rlPhaseEnd
 
-    rlPhaseStartTest "Add agent with tpm_policy generated by create_mb_refstate script"
+    MB_POLICIES=()
+    rlPhaseStarttest "Create measured boot policy using different tools"
         # use installed create_mb_refstate from /usr/share/keylime/scripts
         rlRun "python3 /usr/share/keylime/scripts/create_mb_refstate $NO_SB_PARAM /sys/kernel/security/tpm0/binary_bios_measurements mb_refstate2.txt"
```

Comment:
Would it make sense to do this step also in an `if [ -f /usr/share/keylime/scripts/create_mb_refstate ]; then` block?

Reply:
Or maybe we want to be sure thatwe notice when the script is missing... ?